demos/bootstrap.js

```diff
@@ -0,0 +1,25 @@
+(function() {
+
+	// Find the script element
+	var scripts = document.getElementsByTagName( "script" );
+	var script = scripts[ scripts.length - 1 ];
+
+	// Read the modules
+	var modules = script.getAttribute( "data-modules" );
+	var demoScripts = script.getAttribute( "data-modules" );
+	var path = window.location.pathname;
+	var pathParts = path.split( "/" );
+	var pathLength = pathParts.length;
+	if ( modules ) {
```

Comment:
I'm wondering if we should stick with the pattern we use for tests, where we don't infer the "base" script, but already require at least one module to be specified.